xml/bp_troubleshooting_status.xml

```diff
@@ -866,6 +866,100 @@
+      <para>
+        Resume &cephadm; work with:
+      </para>
+<screen>&prompt.cephuser;ceph orch resume</screen>
+    </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><option>CEPHADM_STRAY_HOST</option></term>
+    <listitem>
+      <para>
+        One or more hosts have running &ceph; daemons but are not registered
+        as hosts managed by &cephadm;. This means that those services cannot
+        currently be managed by &cephadm;. For example, restarted, upgraded, included
+        in <command>ceph orch ps</command>.
+      </para>
+      <para>
+        You can manage the host(s) with:
+      </para>
+<screen>&prompt.cephuser;ceph orch host add *<replaceable>hostname</replaceable>*</screen>
+      <note>
+        <para>
+          You may need to configure SSH access toe hremote host before this will work.
+        </para>
+      </note>
+      <para>
+        Alternatively, you can manually connect to the host and ensure that
+        services on that host are removed or migrated to a host that is managed
+        by &cephadm;.
+      </para>
+      <para>
+        You can also disable this warning entirely with:
+      </para>
+<screen>&prompt.cephuser;ceph config set mgr mgr/cephadm/warn_on_stray_hosts false</screen>
+    </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><option>CEPHADM_STRAY_DAEMON</option></term>
+    <listitem>
+      <para>
+        One or more &ceph; daemons are running but not are not managed by &cephadm;.
+        This may be because they were deployed using a different tool, or because
+        they were started manually. Those services cannot currently be managed
+        by &cephadm;. For example, restarted, upgraded, or included in
+        <command>ceph orch ps</command>..
+      </para>
+      <para>
+        If the daemon is a stateful one (monitor or OSD), it should be adopted
+        by &cephadm;. For stateless daemons, it is usually easiest to provision
+        a new daemon with the <command>ceph orch apply</command> command and
+        then stop the unmanaged daemon.
+      </para>
+      <para>
+        This warning can be disabled entirely with:
+      </para>
+<screen>&prompt.cephuser;ceph config set mgr mgr/cephadm/warn_on_stray_daemons false</screen>
+    </listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><option>CEPHADM_HOST_CHECK_FAILED</option></term>
+    <listitem>
+      <para>
+        One or more hosts have failed the basic &cephadm; host check, which
+        verifies that the host is reachable and cephadm can be executed there,
+        and that the host satisfies basic prerequisites, like a working container
+        runtime (podman or docker) and working time synchronization. If this
+        test fails, cephadm will no be able to manage services on that host.
+      </para>
+      <para>
+        You can manually run this check with:
+      </para>
+<screen>&prompt.cephuser;ceph cephadm check-host *<replaceable>hostname</replaceable>*</screen>
```

Comment:
*'s again, not sure they are menat to be there (maybe yes)

Reply:
```suggestion
<screen>&prompt.cephuser;ceph cephadm check-host <replaceable>hostname</replaceable></screen>
```